inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -87,9 +87,6 @@ weave_peers: uninitialized
 # Set the MTU of Weave (default 1376, Jumbo Frames: 8916)
 weave_mtu: 1376
 
-# Enable kubernetes network policies
-enable_network_policy: false
```

Comment:
why it's removed from here ?

Reply:
we talked with @mattymo and he told that with setting it false calico does not allow anything, though i didn't tested this.

Reply:
this change the default I'm not sure about that, 
I would let the variable in this file, and eventually, switch the value to true? I